### ISSUE TITLE
[Crash] [Music Triggers 1.18.2 - Fabric] Refuses to start if musictriggers.toml is edited

### ISSUE THREAD

**Issue by ShinryuAspect** [TheComputerizer/Music-Triggers#12]
I was about to test some edits in the file when Minecraft decided to crash (I was following the wiki, more precisely, [this](https://github.com/TheComputerizer/Music-Triggers/wiki/The-Basics) and it crashed just as the trigger was expected to play, I assume). Fun fact, I can delete everything from the .toml file and edit it vie a GUI feature, but, as soon as I exit and launch the game again, the crash happens.
Attached crash file. Anything else you may need, just hit me up and I'll do my best to provide it.

[crash-2022-04-11_16.47.21-client.txt](https://github.com/TheComputerizer/Music-Triggers/files/8465129/crash-2022-04-11_16.47.21-client.txt)

**Comment by TheComputerizer** [comment]
Can you give an example of a config you tried making that crashed?

**Comment by ShinryuAspect** [comment]
Sure, implying the song name is mainmenu.ogg, the file is as simple as this: https://pastebin.com/Lk4WwA8H (it doesn't let me upload a toml file, so I'm attaching a pastebin).

**Comment by TheComputerizer** [comment]
Thanks, I'll take a look whenever I wake up

**Comment by TheComputerizer** [comment]
Ill have to do some testing myself with the 1.18.2 fabric versions but Ill push it with the round of fixes for tonight if I find the issue

**Comment by TheComputerizer** [comment]
Ok so I released 5.1 and tested a fairly large config folder without any issues in the 1.18.2 fabric version. You can reopen this if you are still experiencing issues

**Comment by ShinryuAspect** [comment]
Still experiencing the exact same crash as before: Java complaining about an item inside a Map being null when it tries to access it.
I copy-pasted the very first example provided on [The Basics](https://github.com/TheComputerizer/Music-Triggers/wiki/The-Basics) page from the wiki (I made sure the .ogg was also called song1, just like in that example) and that same .0gg file is properly being detected, as it was converted to an .ogg from an .mp3 that it was originally (there are no conversion errors either, as de .ogg plays smoothly and without any problem).
Also, it can't be a mod incompatibility, as I've literally removed every single mod from the folder aside from FabricAPI and MusicTriggers.

[The contents of my musictriggers.toml](https://pastebin.com/wCj56agn) (literally, copied from the examples)
[crash-2022-04-15_21.03.57-client.txt](https://github.com/TheComputerizer/Music-Triggers/files/8497866/crash-2022-04-15_21.03.57-client.txt)

**Comment by TheComputerizer** [comment]
Looks like its a different crash. This one usually happens when the song in your config does not match the name of a sound file in your folder. What is the name of your sound file and is it in the correct place? You could alternatively send a zip of the entire folder so I can just take a look myself

**Comment by ShinryuAspect** [comment]
I renamed the song to be song1 and place it in MusicTriggers/songs/assets/musictriggers/sounds/music
Also, on a side note, if I use the in-game GUI method, I can seem to add the songs and they properly work, until I restart the mod, which then causes this crash.
Also also, it may be mere coincidence. but the one trigger that is causing the crash to me is the "menu" one. I trid to start the game with the "generic" one and it worked, but, as soon as I added the "menu" one, it started to send me that crash. I can add both via the in-game GUI and the "menu" one works, until reset.
